crates/gossip/src/respond.rs

```diff
@@ -155,12 +155,6 @@ impl K2Gossip {
         self.fetch
             .request_ops(decode_ids(accept_response.new_ops), from_peer.clone())
             .await?;
-        self.peer_meta_store
```

Comment:
This was moved to `update_new_ops_bookmark` but didn't get removed here. Not too serious if the received value is accepted but there is a check in the other function. Certainly no need to do it twice